src/test/resources/features/4.0.x.feature

```diff
@@ -0,0 +1,48 @@
+Feature: 4.0.x
+  Scenario Outline: NASA-PDS/validate#<issueNumber>-<subtest>
+    Given validate issue <issueNumber>, test <subtest>, and test data at <datasrc>
+    When execute validate with <args>
+    Then compare to the expected outcome <expectation>.
+    @4.0.x
+    Examples: 
+      | issueNumber | subtest | datasrc | args | expectation |
+
+#| 1408 | 1 | "github1408" | " -t {datasrc}/tc_vl0axrat_table_delimited_valid.xml,{datasrc}/tc_vl0axrat_table_char_valid.xml" | "summary:totalErrors=1,summary:productValidation:failed=1,summary:messageTypes:error.label.table_definition_problem=1" |
```

Comment:
@rchenatjpl I will clean this up, but in the future, for PDS codebases we try to only include commented out code if it is explicitly described and a rationale is given as to why it is included. I am going to remove the skipped tests from here.

If you did test some of these locally, I think the best way to show we accomplished this would be to include those test results in the PR to show they were tested, but could not be included in the automated testing.